frontend/src/components/reports/PendingReportsTab.vue

```diff
@@ -0,0 +1,168 @@
+
+export default {
+  components: { FacilityFilter, CancelSurveyResponseDialog },
+  mixins: [alertMixin],
+
+  props: {
+    loading: {
+      type: Boolean,
+      default: false,
+    },
+    pendingReports: {
+      type: Array,
+      default: () => [],
+    },
+    facilities: {
+      type: Array,
+      default: () => [],
+    },
+  },
+
+  data() {
+    return {
+      headers: [
+        { title: 'Alerts', key: 'alertType' },
+        { title: 'Report ID', key: 'surveyResponseReferenceNumber' },
+        { title: 'Title', key: 'title' },
+        { title: 'Facility', key: 'facilityName' },
+        { title: 'Status', key: 'status' },
+        { title: 'Latest Activity', key: 'latestActivity' },
+        { title: 'Actions', key: 'actions', sortable: false },
+      ],
+      processing: false,
+      showCancelDialog: false,
+      facilityNameFilter: undefined,
+    }
+  },
+
+  computed: {
+    filteredPendingReports() {
+      if (!isEmpty(this.facilityNameFilter)) {
+        return this.pendingReports?.filter((report) => this.filteredFacilityIds?.includes(report.facilityId))
+      }
+      return this.pendingReports
+    },
+
+    filteredFacilityIds() {
+      const filteredFacilities = this.facilities?.filter((facility) => facility.facilityName?.toLowerCase().includes(this.facilityNameFilter?.toLowerCase()))
+      return !isEmpty(filteredFacilities) ? filteredFacilities?.map((facility) => facility.facilityId) : []
+    },
+  },
+
+  methods: {
+    async openSurveyResponse(surveyResponse) {
+      if (surveyResponse?.surveyResponseId) {
+        this.$router.push({ name: 'survey-form', params: { surveyResponseGuid: surveyResponse?.surveyResponseId } })
+      } else {
+        this.createSurveyResponse(surveyResponse)
+      }
+    },
+
+    async createSurveyResponse(surveyResponse) {
+      try {
+        this.processing = true
+        const payload = {
+          contactId: surveyResponse?.contactId,
+          facilityId: surveyResponse?.facilityId,
+          surveyId: SURVEY_IDS.MONTHLY_REPORTING,
+          surveyResponseTitle: surveyResponse?.title,
+          fiscalYearId: surveyResponse?.fiscalYearId,
+          reportingMonthId: surveyResponse?.reportingMonthId,
+          surveyResponseType: this.getSurveyResponseType(surveyResponse?.reportingMonthName),
+        }
+        const response = await ReportsService.createSurveyResponse(payload)
+        this.setSuccessAlert('Started a new report response successfully')
+        if (response?.surveyResponseId) {
+          this.$router.push({ name: 'survey-form', params: { surveyResponseGuid: response?.surveyResponseId } })
+        }
+      } catch (error) {
+        this.setFailureAlert('Failed to start new report response', error)
+      } finally {
+        this.processing = false
+      }
+    },
+
+    getSurveyResponseType(reportingMonthName) {
+      let responseType = SURVEY_RESPONSE_TYPES.MONTHLY
+      if (['November'].includes(reportingMonthName)) {
```

Comment:
I believe these months are just temporary until we hear back from the business? If so can we add a TODO?

Reply:
I've added a comment